app/(articles)/[article].tsx

```diff
@@ -0,0 +1,96 @@
+import {View, Text, Pressable, StyleSheet, ScrollView} from 'react-native';
+import {Image} from 'expo-image';
+import {router, Stack, useLocalSearchParams} from 'expo-router';
+import React from 'react';
+import {WebView} from 'react-native-webview';
+import {FontAwesome5} from '@expo/vector-icons';
+
+export default function ArticleView() {
+  const {title, id = 0} = useLocalSearchParams<{title: string; id?: string}>();
+
+  console.log({title});
+  console.log({id}, id);
+
+  const articleList = [
+    require('../../assets/articleAssets/selfcare/mindfulpause.html'),
+    require('../../assets/articleAssets/selfcare/understandingsleep.html'),
+    require('../../assets/articleAssets/selfcare/sleepqualitychecklist.html'),
+    require('../../assets/articleAssets/understandingyourself/knowyourpersonalitytype.html'),
+    require('../../assets/articleAssets/understandingyourself/identifyandtameyouremotions.html'),
+    require('../../assets/articleAssets/understandingyourself/attachmentstyle.html'),
+    require('../../assets/articleAssets/understandingyourself/distresssigns.html'),
+    require('../../assets/articleAssets/understandingyourself/eustress.html'),
+    require('../../assets/articleAssets/aboutmentalhealth/adhd.html'),
+    require('../../assets/articleAssets/aboutmentalhealth/asd.html'),
+    require('../../assets/articleAssets/aboutmentalhealth/bipolar.html'),
+    require('../../assets/articleAssets/aboutmentalhealth/whatispsychotherapy.html'),
+    require('../../assets/articleAssets/storiesfromothers/helplines.html'),
+  ];
```

Comment:
A lot of these are missing, hence I can't build the project:

https://github.com/safsbe/mental-health-app-assets/tree/9b74e449ed81d0f277ad689c8e5a2141cf0ba0ec/articleAssets

As per-above, are these articles files in scope of this PR? If not, can we remove them?